windrose/windrose_main.py

```diff
@@ -0,0 +1,123 @@
+import os
+import pandas as pd
+import numpy as np
+from windrose import WindroseAnalysis
+import time
+from pathlib import Path
+import argparse
+
+
+class WindroseProcessor:
+    def __init__(self, root_directory, directory='windRoseData'):
+        self.root_directory = Path(root_directory)
+        self.directory = Path(directory)
```

Comment:
What is the difference between root_directory and directory?

Reply:
root_directory represents the top-level folder where the input data files or site folders are stored. 

directory is used to specify or create a subfolder where the output files (processed data) will be saved.